### ISSUE TITLE
[Bug/Feature request] Add compatibility with Translate plugin in Static Pages

### ISSUE THREAD

**Issue by nmiyazaki-chapleau** [wintercms/wn-blocks-plugin#31]
TL;DR: Winter Translate plugin doesn't work with this plugin in Static Pages, because Translate doesn't understand Blocks' multiple levels system. This makes all Translate fields refer to the same field name, and ends up replacing the values of all form fields within blocks that have the same FormField->fieldName.

Given the following block definition:

```
name: block
fields:
    content:
        label: Title
        type: mltext
==
<div>{{content}}</div>
```
Adding two of these blocks to a static page will cause both of them to always have the same value.

Steps:
1. Add this block to a static page twice with different values
2. Save and refresh the page
3. The values of both blocks should be the same as the last block's original value.

This is because Translate will create fields in the forms with the name "RLTranslate[en][content]" for the locale-dependant field.
Furthermore, after saving the default locale's value and refreshing the page, the other locale's value is completely messed up, showing the raw content of the static page of the other locale (static-pages-fr/whatever.htm's content, which is [viewBag] title = "abc" == ).

I have only recently started developping websites using WinterCMS, and tried to fix this for a couple of hours with no success due to my lack of knowledge with its backend / inner workings.

Any kind of fix or workaround would be appreciated, as this is a big blocker for my team right now (even a hardcoded change to a plugin file is fine!).

WinterCMS 1.2.6

**Comment by LukeTowers** [comment]
@mjauvin any thoughts?

**Comment by mjauvin** [comment]
It may eventually be possible to make it work when [this PR](https://github.com/wintercms/wn-translate-plugin/pull/76) is completed.
